src/feature/stats/rephist.c

```diff
@@ -1527,6 +1533,49 @@ rep_hist_load_state(or_state_t *state, char **err)
   return 0;
 }
 
+/** Checks the consensus parameter if BandwidthStatistics is auto,
+ *  and returns 1 if the parameter returned is 1 or not known.
+ *  If BandwidthStatistics is set to true (obviously in torrc) then return 1.
+ *  Otherwise, returns 0. */
+bool
+get_bandwidth_stats_param(void)
+{
+  int bandwidth_stats = get_options()->BandwidthStatistics;
+  /* If bandwidth_stats is auto check the consensus parameter. */
+  if (bandwidth_stats == -1)
+    bandwidth_stats = networkstatus_get_param(NULL,
+                      "BandwidthStatistics", -1, 0, 1);
```

Comment:
We use named constants for consensus parameter defaults, so we can change them easily.
Please see the example function.

Reply:
Important:
If the consensus parameter is not present, the default should be 1.